src/lib/Components/App.php

```diff
@@ -88,7 +94,18 @@ public function jsonSerialize()
                     $this->toolbars,
                     [$this->navigationHub, $this->mainContent]
                 ),
-            ],
+            ];
+        }
+
+        return [
+            'selector' => self::TAG_NAME,
+            'update' => $update,
         ];
     }
+
+    private function isError()
+    {
+        return count($this->notifications) === 1
```

Comment:
The todo list for the PR agrees with you: _Verify that the app exception rendering based on 1 error notification is sufficient and doesn't have unwanted side effects_ :)